docs/source/experiment/overview.rst

```diff
@@ -0,0 +1,15 @@
+NNI Experiment
+==============
+
+An NNI experiment is a unit of one tuning process. For example, it is one run of hyper-parameter tuning on a specific search space, it is one run of neural architecture search on a search space, or it is one run of automatic model compression on user specified goal on latency and accuracy. Usually, the tuning process requires many trials to explore feasible and potentially good-performing models. Thus, an important component of NNI experiment is **training service**, which is a unified interface to abstract diverse computation resources (e.g., local machine, remote servers, AKS). Users can easily run the tuning process on their prefered computation resource and platform. On the other hand, NNI experiment provides **WebUI** to visualize the tuning process to users.
```

Comment:
Could you please explain what is exactly the relation between "experiment" and "training service"? Does the folder "/source/experiment/" only contain docs for training service? I'm a little confused.

Reply:
how about the updated version, I added a figure and one small paragraph. looking for your further comments